### PR TITLE
GCI-349 Add description to readme of openmrs-module-metadatamapping

### PR DESCRIPTION
**Pull request by mee-kell**
Added a concise description to readme.md about metadata mapping, and changed oraclejdk8 to openjdk8 in .travis.yml to pass Travis CI test.

Issue: https://issues.openmrs.org/browse/GCI-349?jql=project%20%3D%20GCI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
 openmrs-module-metadatamapping
-==============================
\ No newline at end of file
+==============================
+
+Provides an easy & explicit way for metadata within a system (encounter types, location, etc.) to be mapped to external vocabularies.
+
+The aim is to solve many of the problems around metadata management by making it easier for developers to refer to metadata, simplify the synchronisation of metadata cross systems, etc. 
+
+For more information, check https://wiki.openmrs.org/pages/viewpage.action?pageId=6619649.
```
Comment (HerbertYiga) [inline README.md]: Ohh thanks for the great work,as i was reviewing through i also found this link that can also be referred too https://wiki.openmrs.org/display/docs/Metadata+Mapping+Module

Comment (mee-kell) [reply README.md]: Thanks for the heads up! I've added that link in now as well.

**Comment by suthagar23** [comment]
You should add more content to the ReadMe file. Please have a look around other ReadMe files in the repository. 
1. Deployment Steps
2. Developer Documentation 
3. License
and etc

**Comment by mee-kell** [comment]
> You should add more content to the ReadMe file. Please have a look around other ReadMe files in the repository.
> 
>     1. Deployment Steps
> 
>     2. Developer Documentation
> 
>     3. License
>        and etc

@suthagar23 Thanks for the advice. I've added a lot more detail now to the file. What do you think?

**Comment by jwnasambu** [comment]
Thanks for adding more content on the ReadMe file however, when I click on the links you shared they don't lead me to those given pages. Kindly fix them.

**Comment by mee-kell** [comment]
@jwnasambu If you were referring to the code style conventions (led to an outdated page), it has been fixed. Thanks for pointing it out. All the other links lead to the intended pages now.

**Comment by Akayeshmantha** [comment]
@suthagar23  @jwnasambu

**Pull request merged**